# Introducing Myself

My name is Kathrin Seibt and I'm a **computational biologist** with a background in molecular biology. My research focus has been on comparative genomics, transposable elements (repetitive DNA sequences), and molecular markers. 

In the laboratory I've been routinely conducting **molecular experiments** including DNA extraction, PCR, electrophoresis, and DNA sequencing - particularly in the context of molecular markers. Due to a high amount of samples I am used to planning, executing, documenting, and interpreting *large-scale experiments*. Besides, I have also applied a wide variety of other laboratory techniques in the fields of molecular biology, microbiology, and analytical chemistry.

I am programming in **Python** and **R** and have a general interest in problem solving, data mining, and visualization. In order to utilize various software tools and integrate command line tools, I am used to working both in a **Windows** and a **Linux** environment. Consequently, I particularly appreciate cross-platform tools. For data visualization and documentation I use common packages such as **matplotlib**, **ggplot2** and **RMarkDown**. 

Due to my previous research on repetitive DNA sequences, I am used to handling **large amounts of data**, design analyses and workflows, as well as apply **analytical** and **rational** criteria in order to solve challenging tasks. In this context, I **enjoy learning** and quickly integrate new technologies. I would describe myself as a curious, **intrinsically motivated** and **open-minded** person. In general, I love to work in a motivated and dedicated **team**, but I also appreciate diving into data **self-dependently**. To be able to reproduce and share my work, I value **documentation** to log workflows, record important resources or tools, and summarize results. I **routinely present** my data in **publications** and **talks** targeting different audiences and also used **social media** ([Twitter](https://twitter.com/kathse1984)) to share publications with the public.

After my career break for **maternity leave** I am now actively **looking for a new position**. I am keen to address challenging tasks, acquire novel skills, integrate new technologies and join a motivated team.

For further information regarding my professional profile, please see my [CV](./CV/).

# Introducing My Visualizations

I enjoy **data mining** and love to create **visualizations** that reveal and illustrate interesting findings. I want to highlight my recently developed cross-platform bioinformatic visualization tool **FlexiDot** available on [GitHub](https://github.com/molbio-dresden/flexidot).

To give you an impression of my work, I want to present some illustrations from my recent publications. 

## 1. Short transposable elements are associated with genes in nightshade plants

The distribution of short interspersed nuclear elements (SINEs, *left*) on the twelve tomato chromosomes is highly similar to the distribution of the annotated genes (*right*).<sup>[1](#myfootnote1)</sup>

<img src="./DataViz/1-Tomato_Transposon_Gene_Heatmap.png" alt="transposon localization in tomato" width="750"/>

Different families of these transposable element (x-axis) are associated to genes to a varying degree in potato (*a*) and, consistently, in the analyzed nightshade plants in general (*b*).<sup>[1](#myfootnote1)</sup>

<img src="./DataViz/2-Transposon_Gene_Association.png" alt="transposon gene association in potato and Solanaceae" width="750"/>

## 2. Visualization tool to illustrate structural sequence features

I developed the cross-platform Python-tool **FlexiDot**<sup>[2](#myfootnote2)</sup> to compare DNA and protein sequences with highly customizable shading and calculation options. FlexiDot is available on [GitHub](https://github.com/molbio-dresden/flexidot)] and according to its Altmetric score it is among the **top 3% of publications** in the **journal Bioinformatics**. I would like to emphasize the detailed **documentation** as supporting material to target a wide range of users. 

<img src="./DataViz/3-FlexiDot.png" alt="FlexiDot feature illustration" width="750"/>

I was able to use our new tool to identify and illustrate modular similarities of short transposon families (Angio-SINEs) across a wide variety plant species.<sup>[2](#myfootnote2)</sup><sup>,</sup><sup>[3](#myfootnote3)</sup> 

<img src="./DataViz/4-Angio_SINE_FlexiDot.png" alt="FlexiDot illustration of Angio-SINEs" width="750"/>

## 3. Comparative genomics analysis of transposable elements in 100 plant genomes

This schematic representation illustrates the modular similarities between the different families of short transposable elements (*right*) and links them to the plant orders they were identified in (*left side*).<sup>[3](#myfootnote3)</sup>

<img src="./DataViz/5-Angio_SINE_Phylogeny.png" alt="phylogenetic distribution of Angio-SINEs" width="750"/>

The different Angio-SINE families vary in the number of copies per genome (*a left, b*), the similarity of the copies within the genome (*a right*) as well as the frequency, in which they are located in annotated genes (*c*).<sup>[3](#myfootnote3)</sup> 

<img src="./DataViz/6-Angio_SINE_Characteristics.png" alt="general characteristics of Angio-SINEs" width="750"/>

# References

## Selected Publications
 
<a name="myfootnote1">1.</a> Seibt, K. M., T. Wenke, K. Muders, B. Truberg & T. Schmidt (2016). *Short interspersed nuclear elements (SINEs) are abundant in Solanaceae and have a family-specific impact on gene structure and genome organization.* **Plant Journal** 86(3): 268-285. [[Weblink](https://doi.org/10.1111/tpj.13170)]

<a name="myfootnote2">2.</a> Seibt, K. M., T. Schmidt & T. Heitkam (2018). *FlexiDot: Highly customizable, ambiguity-aware dotplots for visual sequence analyses.* **Bioinformatics** 34(20): 3575-3577. [[Weblink](https://doi.org/10.1093/bioinformatics/bty395), [GitHub](https://github.com/molbio-dresden/flexidot)]

<a name="myfootnote3">3.</a>  Seibt, K. M., T. Schmidt & T. Heitkam (2019). *The conserved 3′ Angio-domain defines a superfamily of short interspersed nuclear elements (SINEs) in higher plants.* **Plant Journal** 101(3): 681–699. [[Weblink](https://doi.org/10.1111/tpj.14567)]


