# Introducing Myself

My name is Kathrin Seibt and I'm a **bioinformatician** with a background in molecular biology. My research focus has been on comparative genomics, transposable elements (repetitive DNA sequences), and molecular markers.

I'm programming in **Python** and **R** and have a general interest in programming, problem solving, data mining, and visualization. 

Due to my previous research on repetitive DNA sequences, I am used to working with **large amounts of data** and apply **analytical** and **rational** criteria in order to solve challenging tasks. In this context, I **enjoy learning** and integrating new technologies. I would describe myself as an **intrinsically motivated** and **rather introverted** person. In general, I love to work in a motivated and dedicated **team**, but I also appreciate diving into data **self-dependently**.


# Introducing My Visualizations

I enjoy **data mining** and enjoy to create **visualizations** to reveal and illustrate interesting findings. For this purpose, I use both Python and R as well as common packages such as **matplotlib**, **ggplot2** and **RMarkDown**. 

To give you an impression of my work, I want to highlight some illustrations from my recent publications:

## 1. Genomic localization of short transposable elements on tomato chromosomes

The distribution of short interspersed nuclear elements (SINEs) on the twelve tomato chromosomes is highly similar to the distribution of the annotated genes [Ref 1].

![transposon localization in tomato](/DataViz/1-Tomato_Transposon_Gene_Heatmap.png)

## 2. Association of short transposable elements with genes

Different transposable element families (x-axis) are associated to genes to a varying degree in potato (a) and, consistently, in Solanaceae in general (b) [Ref 1].

![transposon gene association in potato and Solanaceae](/DataViz/2-Transposon_Gene_Association.png)

## 3. Visualization tool to illustrate structural sequence features

I developed the cross-platform Python-tool **FlexiDot** to compare DNA and protein sequences with highly customizable shading and calculation options [Ref 2, [GitHub](https://github.com/molbio-dresden/flexidot)]. According to its Altmetric score, FlexiDot is among the **top 3% of publications** in the **journal Bioinformatics**. I would like to emphasize the detailed **documentation** as supporting material to target a wide range of users. 

![FlexiDot feature illustration](/DataViz/3-FlexiDot.png)

## 4. Comparative genomics analysis of transposable elements in 100 plant genomes

FlexiDot illustration of the modular similarities of the Angio-SINE families [Refs 2,3]. 

![FlexiDot Illustration of Angio-SINEs](/DataViz/4-Angio_SINE_FlexiDot.png)

Short transposable elements showing modular similarities (*right*) and their phylogenetic distribution (*left side*)  [Ref 3]. 

![phylogenetic distribution of Angio-SINEs](/DataViz/5-Angio_SINE_Phylogeny.png)

Characterization of the copy number and similarity of all Angio-SINE families in the analyzed plant species [Ref 3]. 

![general characteristics of Angio-SINEs](/DataViz/6-Angio_SINE_Characteristics.png)

# References

> 1. Seibt, K. M., T. Wenke, et al. (2016). *Short interspersed nuclear elements (SINEs) are abundant in Solanaceae and have a family-specific impact on gene structure and genome organization.* **Plant Journal** 86(3): 268-285. [[Weblink](https://doi.org/10.1111/tpj.13170)]
>
> 2. Seibt, K. M., T. Schmidt, et al. (2018). *FlexiDot: Highly customizable, ambiguity-aware dotplots for visual sequence analyses.* **Bioinformatics** 34(20): 3575-3577. [[Weblink](https://doi.org/10.1093/bioinformatics/bty395), [GitHub](https://github.com/molbio-dresden/flexidot)]
>
> 3. Seibt, K. M., T. Schmidt, et al. (2019). *The conserved 3′ Angio-domain defines a superfamily of short interspersed nuclear elements (SINEs) in higher plants.* **Plant Journal** 101(3): 681–699. [[Weblink](https://doi.org/10.1111/tpj.14567)]


